identity.js

```diff
@@ -4,13 +4,21 @@
  * Copyright (c) 2015-2016 Digital Bazaar, Inc. All rights reserved.
  *
  */
-define(['angular', './identity-service'], function(angular, identityService) {
+define([
+  'angular',
+  './identity-service',
+  './identity-component',
+  './identity-settings-component'],
+  function(angular, identityService) {
 
 'use strict';
 
 var module = angular.module('bedrock.identity', []);
```

Comment:
the array here should contain all the Angular dependencies used by all the components in this modules, 'bedrock.alert', 'bedrock.resource', 'bedrock.session' etc.